service/src/main/java/com/ge/predix/acs/privilege/management/PrivilegeManagementServiceImpl.java

```diff
@@ -290,6 +297,23 @@ public BaseSubject getBySubjectIdentifier(final String subjectIdentifier) {
         return subject;
     }
 
+    @Transactional(readOnly = true)
```

Comment:
getSubjects() method never refers to subjectScopedAccessRepository. It has to use this when titan is enabled.